### DISCUSSION TITLE
Release qibb v1.11.0

### DISCUSSION THREAD

**Discussion by vkyosev** [techtriq/qibb#24] [Announcements]
_v1.11.0 was published on 5 July 2022._

### Summary

* Related Flows Card
  * While browsing the catalog, you can now more easily discover what flows are using a particular node.
  * You can find the new card in the overview page of a node. It lists up to 5 flows which are containing the node and allow you to quickly discover the details by clicking on them.

* Various improvements and fixes
  * The overview page of a node now offers a “How to install” section to give users some guidance.
  * To offer a better developer experience, we have added more details to the OpenAPI of the Flow Catalog API, such as returned types. 
  * A new set of API Paths for the Flow Catalog API, which deprecate the old paths.
  * We have changed the arrangement of the elements in the Navigation Sidebar to move the Catalog closer to the top.
  * Added better UI feedback for buttons.
  * The header of the catalog is now automatically resizing to a compact size while scrolling through the list of items.

<hr /><em>This discussion was created from the release <a href='https://github.com/techtriq/qibb/releases/tag/v1.11.0'>v1.11.0</a>.</em>

**Comment by rholzhause** [comment]
Another one! 🎉🎉🎉 That's great!

**Reply by TristanRiedel** [reply]
Can't wait for the next one! 😎
